src/commands/archive-setup/archive.setup.ts

```diff
@@ -0,0 +1,91 @@
+import { fsa } from '@chunkd/fs';
+import { command, option, optional, positional } from 'cmd-ts';
+
+import { CliInfo } from '../../cli.info.ts';
+import { logger } from '../../log.ts';
+import { config, registerCli, S3Path, tryParseUrl, UrlFolder, urlToString, verbose } from '../common.ts';
+
+export const archiveSetup = command({
+  name: 'archive-setup',
+  description: 'Verify the if the source path car be archived and output the archive location to use',
```

Comment:
```suggestion
  description: 'Verify if the source path can be archived and output the archive location to use',
```
(and re-run docs generation)